File: ABC/ABC346/C/main.cc
#include <iostream>
#include <set>

using namespace std;
using ll = long long;

int main() {
  int n;
  ll k;
  cin >> n >> k;

  set<ll> se;
  ll res = k * (k + 1) / 2;
  while (n--) {
    ll a;
    cin >> a;
    if (a <= k && se.insert(a).second) {
      res -= a;
    }
  }

  cout << res << endl;
  return 0;
}

File: ABC/ABC346/C/main.py
n, k = map(int, input().split())
a = list(map(int, input().split()))

se = set()
res = k * (k + 1) // 2
for b in a:
    if b <= k and not b in se:
        res -= b
    se.add(b)

print(res)